inc/saiobject.h

```diff
@@ -58,37 +58,43 @@ typedef struct _sai_object_key_t
 /**
```

Comment:
@AshokDaparthi considering the proposed approach it seems that the current model assumes the host adapter needs to maintain some sort of object id storage per switch-id, to be reconciled post warm restart. is there a way to query the number of switch-ids supported post warm restart?